src/components/home/about/imageList.js

```diff
@@ -0,0 +1,29 @@
+// need to replace with proper photos
+export const imageList = [
+  {
+    image:
```

Comment:
Do we have actual proper images to replace these with? They're good cats but I don't think they're CODE PDX volunteers.

Reply:
I will push images to this branch. For now it will just be 5 people, in a future PR they will be several members and it will randomly select and load 5.

Also font size will be smaller than the text if it becomes h6. It I do have an xl Iphone but it looks fine on my phone. Its the same size as the rest of the headings so It can just stay as is. https://codeforpdx.github.io/ you have to click the icon to route to home and you can see how it looks (I'm adding the home link to navbar soon)